packages/nimble-components/src/table-column/date-text/models/format-helper.ts

```diff
@@ -0,0 +1,16 @@
+import type { TableNumberFieldValue } from '../../../table/types';
+
+export function formatNumericDate(
+    formatter: Intl.DateTimeFormat,
+    date: TableNumberFieldValue
+): string {
+    if (typeof date === 'number') {
+        try {
+            return formatter.format(date);
+        } catch (e) {
+            return '';
```

Comment:
It will throw if you pass it something it can't format, like `'foo'` or a value which is out of range for a `Date`. I have tests for a whole range of bad input values that ensure we render a blank cell (near the top of the spec file).